etc/picongpu/titan-ornl/picongpu.profile.example

```diff
@@ -37,7 +37,7 @@ export FC=$(which ftn)
 
 # required tools and libs
 module load git
-module load cmake3/3.9.0
+module load cmake3/3.10.0  # missing
```

Comment:
QUESTION:
Do you want to commit using a missing module?

Reply:
it doesn't matter, both won't work and someone looking at it will see to use this version or any newer